package.json

```diff
@@ -32,7 +32,10 @@
     "ts-jest": "^29.0.3",
     "tsup": "^6.5.0",
     "turbo": "^1.6.3",
-    "typescript": "4.9.4"
+    "typescript": "4.9.4",
+    "next": "^13.0.2",
+    "react": "18.2.0",
+    "react-dom": "18.2.0"
```

Comment:
❓ why did we add them to the root level package.json?

Reply:
Hey @dimkl , I added them at the root since we need these dependencies while developing but we don't want them installed as dependencies of our packages. 

Keep in mind that this is a test PR and we will possibly revert the changes once testing is completed.